FrameworkInternals/original_files.txt

```diff
@@ -108,6 +108,9 @@ File designToConfigValidator.jinja          must_exist,must_be_versioned, md5=ch
 File designToConfigurationXSD.jinja         must_exist,must_be_versioned, md5=check, install=overwrite
 File designToConfigurator.jinja             must_exist,must_be_versioned, md5=check, install=overwrite
 
+Directory Configuration/include install=create
```

Comment:
FYI Configuration module is slightly special in that it includes *directly* from Configuration rather than from Configuration/include. If you insist on putting that file in Configuration/include then you should fix the master CMakeLists.

Reply:
I put the new file in Configuration/include as it's user facing: Configuration decoration code must include that file and I felt it better reinforced the quasar norm of putting include files in _include_ dir.

What do you think? Have a strong preference ?

Reply:
A-ha I checked and Configuration/include is already included by the master CMakeLists even though it did not exist so far.
OK, I agree, it could be added there.